src/main.rs

```diff
@@ -250,7 +250,7 @@ fn project_process(matches: &ArgMatches) -> Result<String, String> {
     let config = fetch_config(matches)?;
     let project = fetch_project(matches, config.clone())?;
 
-    projects::next_item_interactive(config, &project)
+    projects::process_items_interactive(config, &project)
```

Comment:
Good switch, what do you think of process_items instead?

The interactive part of the naming is a holdover from my previous API and might not be valuable any more.

Reply:
sure, will rename it in a bit